frontend/src/components/shared/FeedbackDialog.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { FormEvent, useState } from 'react';
+import Dialog, { DialogProps } from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { withStyles } from '@material-ui/core/styles';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Rating from '@material-ui/lab/Rating';
+import { Locale } from 'scenes/types';
+import { Connection } from 'twilio-client';
+import { PrimaryButton } from './RoundedButton';
+
+interface FeedbackDialogProps extends DialogProps {
+  connection: Connection;
+  locale: Locale;
+}
+
+// Reference for Problem Keys: https://www.twilio.com/docs/voice/sdks/javascript/overview-1x-deprecated/connection
+const EN_STRINGS = {
+  FEEDBACK_DIALOG_TITLE: 'How was your call?',
+  FEEDBACK_SUBMIT_LABEL: 'Submit',
+  FEEDBACK_TITLES: {
+    GOOD: 'GOOD',
+    BAD: 'BAD',
+  },
+  PROBLEM_TITLES: {
+    'Call got disconnected': Connection.FeedbackIssue.DroppedCall,
+    'Lag in audio': Connection.FeedbackIssue.AudioLatency,
+    "Can't hear one another": Connection.FeedbackIssue.OneWayAudio,
+    'Sound breaking up': Connection.FeedbackIssue.ChoppyAudio,
+    'Poor audio quality': Connection.FeedbackIssue.NoisyCall,
+    'Audible echos during call': Connection.FeedbackIssue.Echo,
```

Comment:
I wonder if this is understandable to users. I'm not saying we should change it, but we can keep an eye out.

Reply:
Yeah the copywriting on the left side can be improved haha. Do you have better ideas?

Reply:
No, I don't really have better ideas. I think your suggestions are pretty reasonable. Let's just keep an eye on this.